Libraries/Image/RCTImageLoader.m

```diff
@@ -390,31 +401,6 @@ - (RCTImageLoaderCancellationBlock)loadImageOrDataWithURLRequest:(NSURLRequest *
       request = mutableRequest;
     }
 
-    // Check for cached response before reloading
-    // TODO: move URL cache out of RCTImageLoader into its own module
-    NSCachedURLResponse *cachedResponse = [_URLCache cachedResponseForRequest:request];
-
-    while (cachedResponse) {
-      if ([cachedResponse.response isKindOfClass:[NSHTTPURLResponse class]]) {
-        NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *)cachedResponse.response;
-        if (httpResponse.statusCode == 301 || httpResponse.statusCode == 302) {
```

Comment:
I heard that redirect response cannot arrive as ```NSURLResponse```, and I tested and checked it.
And by [line 374](https://github.com/facebook/react-native/pull/8235/files#diff-905836ab11c0592e87588675db52ee8bL374), that code doesn't process 301 or 302. I never heard the ```NSURLResponse``` whit ```GET``` method has status code 301 or 302. If it's wrong, please let me know.